examples/plutus-transfer/src/Cardano/PlutusExample/Transfer.hs

```diff
@@ -35,10 +36,12 @@ transfer :: TransferParams -> Contract (Last Text) TransferSchema Text ()
 transfer (TransferParams outputPerTx allPayments) = do
   tell $ Last $ Just "Contract started"
   let txs =
-        map (mconcat . map (uncurry Constraints.mustPayToPubKey . first PaymentPubKeyHash)) $
-          group outputPerTx allPayments
+        map toTx $ group outputPerTx allPayments
   forM_ txs $ \tx -> submitTx tx >> waitNSlots 1
   tell $ Last $ Just "Finished"
+  where
+    toTx :: [(PubKeyHash, Value)] -> TxConstraints Void Void
+    toTx = mconcat . map (uncurry Constraints.mustPayToPubKey . first PaymentPubKeyHash)
```

Comment:
Theres mconcatMap in Data.List.Extra, but unsure if its worth it. Happy to merge as is